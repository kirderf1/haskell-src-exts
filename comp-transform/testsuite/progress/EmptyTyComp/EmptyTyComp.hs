{-# LANGUAGE ComposableTypes #-}

piececategory A

type B = A ==> ()

main = putStrLn $ "Det funkar!"
