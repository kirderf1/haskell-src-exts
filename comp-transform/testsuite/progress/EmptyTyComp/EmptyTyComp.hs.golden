ParseFailed (SrcLoc "comp-transform/testsuite/progress/EmptyTyComp/EmptyTyComp.hs" 5 17) "Parse error: )"
